rts/motoko-rts-tests/src/closure_table.rs

```diff
@@ -1,14 +1,14 @@
 use crate::memory::TestMemory;
 
-use motoko_rts::closure_table::{closure_count, recall_closure, remember_closure};
+use motoko_rts::continuation_table::{closure_count, recall_closure, remember_closure};
 use motoko_rts::types::{SkewedPtr, Words};
 
 pub unsafe fn test() {
     println!("Testing closure table ...");
 
     assert_eq!(closure_count(), 0);
 
-    const N: usize = 2000; // >256, to exercise `double_closure_table`
+    const N: usize = 2000; // >256, to exercise `double_continuation_table`
```

Comment:
What is `double_continuation_table`?

Reply:
Not on my dev computer right now so can't check, but there should be some code path for doubling the size of closure table. This number was chosen to exercise that. I guess there isn't a function with that name anymore, this should just say "> 256 to exercise growing continuation table".

Reply:
Ah, I see:
```
$ git grep double_continuation_table
rts/motoko-rts-tests/src/continuation_table.rs:    const N: usize = 2000; // >256, to exercise `double_continuation_table`
rts/motoko-rts/src/continuation_table.rs:unsafe fn double_continuation_table<M: Memory>(mem: &mut M) {
rts/motoko-rts/src/continuation_table.rs:        double_continuation_table(mem);
```